packages/scripts/src/getConversationText.ts

```diff
@@ -0,0 +1,41 @@
+import { MongoClient, BSON } from "mongodb";
+import { assertEnvVars } from "mongodb-rag-core";
+import { Conversation } from "mongodb-chatbot-server";
+
+import "dotenv/config";
+
+const { MONGODB_DATABASE_NAME, MONGODB_CONNECTION_URI } = assertEnvVars({
+  MONGODB_DATABASE_NAME: "",
+  MONGODB_CONNECTION_URI: "",
+});
+
+const conversationIdString = process.argv[2];
+if (!conversationIdString) {
+  throw new Error(
+    "Missing conversation id. Usage: getConversationText <conversationId>"
+  );
+}
+
+const conversationId = new BSON.ObjectId(conversationIdString);
+
+(async function main() {
+  const client = await MongoClient.connect(MONGODB_CONNECTION_URI);
+  try {
+    const db = client.db(MONGODB_DATABASE_NAME);
+    const conversation = await db
+      .collection<Conversation>("conversations")
+      .findOne({ _id: conversationId });
+
+    if (!conversation) {
+      throw new Error(`Conversation ${conversationId} not found`);
+    }
+
+    const converationTranscript = conversation.messages
+      .filter((message) => ["user", "assistant"].includes(message.role))
+      .map((message) => `${message.role.toUpperCase()}: ${message.content}`)
+      .join("\n\n");
+    process.stdout.write(converationTranscript);
```

Comment:
why this instead of console.log?

Reply:
Oh didn't mean to leave this in. Ultimately I want to be able to run the script and pipe the results from stdout into a file, e.g.

```
npm run scripts:getConversationText -- 65aaf4dd36a5cac6e828b6be >> conversation-text.txt
```

But that doesn't really work because we have a bunch of other stuff in stdout. Most of it I think we can suppress but right now there's no way to suppress our `rag-core` logger which prevents us from doing this.

Reply:
i think you could suppress the logger! we do that in the test suites. 

something like:

```ts
import { logger } from "mongodb-rag-core";
// silence logger for tests
logger.transports.forEach((t) => (t.silent = true));
```